tools/webpack-core/package.json

```diff
@@ -14,7 +14,7 @@
   "main": "lib/plugins/index.js",
   "scripts": {
     "compile": "npm run build",
-    "build": "tsc 1>&2",
+    "build": "tsc 1>&2 && cpx \"./src/utils/resolve-recurse/*\" \"./lib/utils/resolve-recurse",
```

Comment:
If we just set [allowJs](https://www.typescriptlang.org/tsconfig#allowJs) in our tsconfig, would we be able to skip this copy step?

Reply:
I just added allowJs: true in the tsconfig for webpack-core. I tried it and the resolve-recurse folder does not show up in lib.

Reply:
Maybe you need to change the `require("../utils/resolve-recurse/resolve.js");` to an `import` too?

Reply:
It wants types if we do that. "Could not find a declaration file for module '../utils/resolve-recurse/resolve.js'. 'd:/imodeljsgit/tools/webpack-core/src/utils/resolve-recurse/resolve.js' implicitly has an 'any' type." Should I look into making types for it? I'm not entirely sure how to do so

Reply:
Yeah, it shouldn't be too hard to make a .d.ts file for it.  Although it may be just as easy to convert the file to TypeScript... 😄

Reply:
Okay I gave the .d.ts file a shot. Theres probably a better way to write it, so let me know if you have any criticisms